frontend/src/views/ProfileView.vue

```diff
@@ -18,19 +21,44 @@ const logout = async () => {
 
   router.push('/')
 }
+
+/**
+ * Perform a search based on the bookmark id.
+ *
+ * If a route is found for the search term then redirect to that route.
+ * Otherwise log an error.
+ *
+ * @param geneSymbol Gene symbol to search for
+ */
+const performSearch = async (geneSymbol: string) => {
+  const routeLocation: any = await search(geneSymbol, 'grch37')
+  if (routeLocation) {
+    router.push(routeLocation)
+  } else {
+    console.error(`no route found for ${geneSymbol}`)
+  }
+}
```

Comment:
Is this really gene symbol? Not rather the object identifier / small / large variant?